EllisWeb.Gematria/Calculator.cs

```diff
@@ -167,50 +165,89 @@ public static long GetNumericGematriaValue(string sourceString, GematriaType gem
         /// A value of 15 will always be represented as ט"ו and 16 will be represented as ט"ז, following Jewish custom. 
         /// </remarks>
         /// <returns>Gemtria Numeric representation of given number</returns>
-        public static string ConvertToGematriaNumericString(long number, bool includeSeparators = true, char thousandsSeparator = '\'', char tensSeparator = '"')
```

Comment:
please keep the original signature as an overload, and just generate a `GematriaOptions` object there using the values provided. No need to break this for people unnecessarily.

Reply:
ok.